pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -535,6 +542,10 @@ func doNetworkInstanceCreate(ctx *zedrouterContext,
 		}
 	default:
 	}
+	// setup the ACLs for the bridge
```

Comment:
Why didn't we need this before?

Note that the bridge and network instance is created before any app can use it.

Reply:
I understand, however, in the existing implementation, there are the mangle table/loging-drop rules, which are currently tied with the bridge interface, on AppNetworkCreation  path (createACLConfiglet) . And they do not have the VIf/VIp associated with them. And with multiple app instances tied to the same network instance, we will end up in creating the duplicate rules. This can be avoided, if we can move these mangle rules to the bridge CRUD operation context. With this motivation I moved these magle/drop rules to network instance creation context.

Reply:
Thanks for clarifying. And with the reordered diffs I can finally see that you broke those out.